mapk_demon/transmitters.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import division
+
+from functools import partial
+
+import pyro
+from pyro import sample
+from pyro.distributions import Normal, Uniform
+from torch import tensor
+
+
+def g1(a):
+    return a / (a + 1)
+
+
+def g2(a):
+    return a ** 2 / (a ** 2 + a + 1)
+
+
+def f(mu, N):
+    return N * mu ** (0.5) + mu
+
+
+hyperparameters = {
+    "t_3k": 1.2,
+    "t_2k": 1.2,
+    "t_k": 0.003
+}
+
+params = {
+    "alpha_3k": 500,
+    "alpha_2k": 500,
+    "alpha_k": 500,
+    "nu_3k": .15,
+    "nu_2k": .15,
+    "nu_k": .15
+}
+
+
+def f_map3k(E1, N_3k):
+    total_3k = hyperparameters['t_3k']
+    alpha_3k = params['alpha_3k']
+    nu_3k = params['nu_3k']
+
+    map3k_mu = total_3k * g1(E1 * (alpha_3k/nu_3k))
```

Comment:
This is good, you can use the original parameters and not the `beta_3k` here, because you just want to generate, not do inference with this model.